test/Flows.test.ts

```diff
@@ -92,13 +92,14 @@ options.forEach(params => {
         await rhub.depositFor(paymaster.address, { value: (1e18).toString() })
 
         relayClientConfig = {
-          logLevel: 'error',
+          logLevel: 'debug',
```

Comment:
I don't want tests to output thousands of lines of useless logs by default. I prefer having 'error' level checked-in, and setting it to 'debug' when, well, actually debugging the test.

Reply:
Mistake.. Should have added //NOGIT locally to prevent myself from committing it